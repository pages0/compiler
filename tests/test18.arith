let x = 5 in let y = 3 in ( x > y or x =y)