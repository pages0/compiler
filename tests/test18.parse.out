( let [ x 5 ] ( let [ y 3 ] ( or ( > x y ) ( = x y ) ) ) )